parsers/getter/brent.py

```diff
@@ -10,7 +10,10 @@
 def format_string(date_string):
     return datetime.strptime(date_string, "%Y%m%d").strftime("%Y-%m-%d")
 
-
+# MS: can't understand why we have assert inside a codebase ?
+# EP: please mark your comments initials, so that it is tracable who is writing 
+#     the reason for this assert is that it should be converted to test
+#     better way os saying that is TODO(MS): convert to test
```

Comment:
@epogrebnyak got it. I did the test for this here https://github.com/mini-kep/parsers/pull/27/files#diff-420a956e849c55a144dfe7e6f34d13acR54

I will remove this comments in code, And assert too.